.changes/fix-set-window-effects-main-thread.md

```diff
@@ -0,0 +1,18 @@
+---
+"@tauri-apps/api": "patch:bug"
+"tauri-utils": "patch:bug"
+"tauri-macos-sign": "patch:bug"
+"tauri-bundler": "patch:bug"
+"tauri-runtime": "patch:bug"
+"tauri-runtime-wry": "patch:bug"
+"tauri-codegen": "patch:bug"
+"tauri-macros": "patch:bug"
+"tauri-plugin": "patch:bug"
+"tauri-build": "patch:bug"
+"tauri": "patch:bug"
+"@tauri-apps/cli": "patch:bug"
+"tauri-cli": "patch:bug"
+"tauri-driver": "patch:bug"
+---
+
+Fixed set_window_effects not runs on main thread in WindowBuilder.
```

Comment:
Chill, the one from 584c3d86c05974815dad4fcb8af8ccd3dab1a4c7 was good enough, we don't need to do this to all packages

```suggestion
---
"tauri": "patch:bug"
---

Fixed set_window_effects not runs on main thread in WindowBuilder.
```